MedicalDocManagment.WebUI/app/components/childrens-card/child-card/neurologists-examination/form.component.html

```diff
@@ -0,0 +1,502 @@
+            <input type="text" class="form-control" id="neurologicalStateWalking"
+                   name="neurologicalStateWalking"
+                   [(ngModel)]="_neurologistsExamination.neurologicalState.walking"
+                   #neurologicalStateWalking="ngModel"
+                   [disabled]="_isSaving" maxlength="93" />
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- Гіперкінези: -->
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="form-group">
+          <div class="input-group">
+            <span class="input-group-addon">Гіперкінези:</span>
+            <input type="text" class="form-control" id="neurologicalStateHyperkinesis"
+                   name="neurologicalStateHyperkinesis"
+                   [(ngModel)]="_neurologistsExamination.neurologicalState.hyperkinesis"
+                   #neurologicalStateHyperkinesis="ngModel"
+                   [disabled]="_isSaving" maxlength="87" />
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- Епісиндром -->
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="form-group">
+          Епісиндром:
+          <textarea class="form-control" id="neurologicalStateEpisyndrome"
+                    name="neurologicalStateEpisyndrome"
+                    [(ngModel)]="_neurologistsExamination.neurologicalState.episyndrome"
+                    #neurologicalStateEpisyndromeTextArea="ngModel"
+                    [disabled]="_isSaving" maxlength="286" rows=3></textarea>
+        </div>
+      </div>
+    </div>
+    <!-- Функції тазових органів -->
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="form-group">
+          <div class="input-group">
+            <span class="input-group-addon">Функції тазових органів</span>
+            <input type="text" class="form-control" id="neurologicalStateFunctionsOfPelvicOrgans"
+                   name="neurologicalStateFunctionsOfPelvicOrgans"
+                   [(ngModel)]="_neurologistsExamination.neurologicalState.functionsOfPelvicOrgans"
+                   #neurologicalStateFunctionsOfPelvicOrgans="ngModel"
+                   [disabled]="_isSaving" maxlength="76" />
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- ==== End of neurological state group ==== -->
+
+    <br />
+    <!-- Діагноз -->
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="form-group">
+          <b>Діагноз:</b> 
+          <diagnosis-text [diagnosisId]="_neurologistsExamination.diagnosisId"></diagnosis-text>
+          <br />
+          <diagnosis-select-modal [(diagnosisId)]="_neurologistsExamination.diagnosisId">
+          </diagnosis-select-modal>
+          <button type="button" class="btn btn-primary" [disabled]="_isSaving" 
+                  data-toggle="modal" data-target="#diagnosisSelectModal">
+            <span *ngIf="_neurologistsExamination.diagnosis">Змінити</span>
```

Comment:
This span never shows up, neither after choosing diagnosis nor after saving complete examination. I will confirm it by attaching screenshot.